tools/wpt/testfiles.py

```diff
@@ -252,6 +260,14 @@ def affected_by_wdspec(test):
                     break
         return affected
 
+    def affected_by_interfaces(file_contents):
+        if 'idlharness.js' in file_contents:
+            for interface in interfaces_changed_names:
+                if '\'' + interface + '\'' in file_contents \
```

Comment:
This will work for `idl_test`, but tests still fetching /interfaces/something.idl would not be considered affected. Maybe also look for `interface + '.idl'`?